ci/ray_ci/automation/filter_tests.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+from ci.ray_ci.utils import query_all_test_names_by_state
+from ray_release.configs.global_config import init_global_config
+from ray_release.bazel import bazel_runfile
+import sys
+
+
+def main():
+    """
+    Filter tests based on test targets and test state.
+    Read list of test targets from file path.
+    Write back into the same file path with flaky tests removed.
+
+    Args:
+        test_targets_file_path: Path to file containing list of test targets.
+        test_state: Test state to filter by.
+            Use string representation from ray_release.test.TestState class.
+    """
+    # Initialize global config
+    init_global_config(bazel_runfile("release/ray_release/configs/oss_config.yaml"))
+
+    # Process arguments
+    test_targets_file_path = None
```

Comment:
feels that we do not need to init it here?

Reply:
It requires to init global config to use `Test.gen_from_s3` I think. I got an error for not init. 
```
Traceback (most recent call last):
  File "/private/var/tmp/_bazel_kevin/dab4782df54f6d33eb80c52e3542c00c/execroot/com_github_ray_project_ray/bazel-out/darwin_arm64-opt/bin/ci/ray_ci/automation/filter_tests.runfiles/com_github_ray_project_ray/ci/ray_ci/automation/filter_tests.py", line 34, in <module>
    main()
  File "/private/var/tmp/_bazel_kevin/dab4782df54f6d33eb80c52e3542c00c/execroot/com_github_ray_project_ray/bazel-out/darwin_arm64-opt/bin/ci/ray_ci/automation/filter_tests.runfiles/com_github_ray_project_ray/ci/ray_ci/automation/filter_tests.py", line 27, in main
    filtered_test_targets = omit_tests_by_state(
                            ^^^^^^^^^^^^^^^^^^^^
  File "/private/var/tmp/_bazel_kevin/dab4782df54f6d33eb80c52e3542c00c/execroot/com_github_ray_project_ray/bazel-out/darwin_arm64-opt/bin/ci/ray_ci/automation/filter_tests.runfiles/com_github_ray_project_ray/ci/ray_ci/utils.py", line 136, in omit_tests_by_state
    state_test_names = query_all_test_names_by_state(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/var/tmp/_bazel_kevin/dab4782df54f6d33eb80c52e3542c00c/execroot/com_github_ray_project_ray/bazel-out/darwin_arm64-opt/bin/ci/ray_ci/automation/filter_tests.runfiles/com_github_ray_project_ray/ci/ray_ci/utils.py", line 105, in query_all_test_names_by_state
    tests = Test.get_tests(prefixes)
            ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/var/tmp/_bazel_kevin/dab4782df54f6d33eb80c52e3542c00c/execroot/com_github_ray_project_ray/bazel-out/darwin_arm64-opt/bin/ci/ray_ci/automation/filter_tests.runfiles/com_github_ray_project_ray/release/ray_release/test.py", line 157, in get_tests
    tests.extend(Test.gen_from_s3(test_prefix))
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/var/tmp/_bazel_kevin/dab4782df54f6d33eb80c52e3542c00c/execroot/com_github_ray_project_ray/bazel-out/darwin_arm64-opt/bin/ci/ray_ci/automation/filter_tests.runfiles/com_github_ray_project_ray/release/ray_release/test.py", line 128, in gen_from_s3
    bucket = get_global_config()["state_machine_aws_bucket"]
             ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: 'NoneType' object is not subscriptable
```